doc/nrf/installation/install_ncs.rst

```diff
@@ -124,7 +124,7 @@ Depending on your preferred development environment, complete the following step
       .. parsed-literal::
          :class: highlight
 
-         nrfutil toolchain-manager config --set toolchain-index=*custom_toolchain_URL*
+         nrfutil toolchain-manager config --set toolchain-index=\*custom_toolchain_URL\*
```

Comment:
Still missing an empty space after the backslash. The latter backslash is probably not needed.